.buildkite/pipelines/chrome_forward_testing.yml

```diff
@@ -117,15 +117,57 @@ steps:
         - exit_status: '-1'
           limit: 1
 
-  - command: .buildkite/scripts/steps/functional/security_serverless_rule_management_prebuilt_rules.sh
-    label: 'Serverless Rule Management - Prebuilt Rules - Security Solution Cypress Tests'
+  - command: .buildkite/scripts/steps/functional/security_serverless_rule_management_prebuilt_rules_customization.sh
```

Comment:
Note: I think the cypress runner is actually not prepared for testing against chrome beta - at least not for now. I can't find where in the cypress config we prescribe the browser, but that should respect the `USE_CHROME_BETA` env variable. I'll approve this for now, because this PR is not concerning already existent states